### PR TITLE
Install langchainrb + ArticleTool

### PR DESCRIPTION
**Pull request by andreibondarev**
This PR installs `langchainrb` and illustrates how to build custom Tools.

Usage examples:
```ruby
# Instantiate the LLM
llm = Langchain::LLM::GoogleVertexAI.new(project_id: ENV["GOOGLE_VERTEX_AI_PROJECT_ID"], region: "us-central1")
# Create a new Thread to keep track of the messages
thread = Langchain::Thread.new

# Instantiate tools
news_retriever = Langchain::Tool::NewsRetriever.new(api_key: ENV["NEWS_API_KEY"])
article_tool = ArticleTool.new

assistant = Langchain::Assistant.new(
  llm: llm,
  thread: thread,
  instructions: "You are a News Assistant.",
  tools: [news_retriever, article_tool]
)

assistant.add_message_and_run content:"What are the latest news from Google I/O?", auto_tool_execution: true

# Inspect the messages
assistant.thread.messages

assistant.add_message_and_run content:"Save the last one to the database", auto_tool_execution: true
```

### CODE REVIEW

File: webapp/.tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```
Comment (andreibondarev) [inline webapp/.tool-versions:1]: This file is used by `asdf`, runtime manager that I use. You don't need to use it.

**Comment by andreibondarev** [comment]
@palladius I'm still trying to figure out why an error is happening...

**Comment by palladius** [comment]
Thanks Andrei!!!

File: webapp/.tool-versions (hunk 1 above)
Comment (palladius) [reply webapp/.tool-versions:1]: Thanks, I'm definitely gonna explore ASDF soon too.

**Comment by palladius** [comment]
I also get error.

**Pull request merged**